Dockerfile.processing_image

```diff
@@ -12,11 +12,12 @@ ENV PIP_NO_CACHE_DIR=1
 # HACK: Create a no-op npm; npm build not needed for backend-only operations
 # The no-op npm touches some files that are the cellxgene install script expects to exist
 RUN git clone https://github.com/chanzuckerberg/single-cell-explorer.git \
- && echo '#!/bin/sh\nmkdir -p build/static && touch build/static/.gitignore build/index.html build/csp-hashes.json' > /usr/bin/npm && chmod 755 /usr/bin/npm \
- && cd single-cell-explorer \
- && make build-for-server-dev \
- && cd .. \
- && rm -rf single-cell-explorer client build/client /usr/bin/npm
+  && echo '#!/bin/sh\nmkdir -p build/static && touch build/static/.gitignore build/index.html build/csp-hashes.json' > /usr/bin/npm && chmod 755 /usr/bin/npm \
+  #  NOTE: Uncomment this line for prod deployment
+  # && cd single-cell-explorer \
```

Comment:
Actually these two lines lol

Reply:
also the `cd ..` line below?

Reply:
actually I think the next line will be interpreted as a new command and the `&&` will cause it to fail, being an invalid start of a command.  The trailing `\` need to be continuous on every line that is connected; the comments break that.

Reply:
ah interesting, the `cd ..` was already there, so maybe I need to add another `cd ..` and comment it out 🤔 ?

Reply:
oh wait, I confused myself 😭 The commented code should be checking out `staging` ahaha

Reply:
![Uploading Screen Shot 2021-11-03 at 12.43.19 PM.png…]()